### PR TITLE
update association page prior to annual meeting

### PR DESCRIPTION
**Pull request by rantahar**
- Rewrite a few sentences to match current status of the assocation.
- Add Finnish translation of the statutes.

### CODE REVIEW

File: content/association/_index.md [hunk 1]
```diff
@@ -5,37 +5,119 @@ title = "Association"
...
+Nordic Research Software Engineers Ry.
 Registering as an association comes with certain legal benefits, such as the ability to
 sign contracts and own assets.
 Registering an association will also create a clear membership and governing
 procedures.
 
-The association is governed by an annual assembly of its members, which is held
-mainly online, and its day to day activities are taken care of by a board.
+The association is governed by an annual assembly of it's members, which is held
+mainly online, and it's day to day activities are taken care of by a board.
```
Comment (bast) [inline content/association/_index.md:21]:
```suggestion
mainly online, and its day to day activities are taken care of by a board.
```

Comment (bast) [reply content/association/_index.md:21]: Also the other: it should be "its". GitHub does not let me line-suggest the other one.

File: content/association/_index.md [hunk 2]
```diff
@@ -5,37 +5,119 @@ title = "Association"
...
-The Nordic RSE group is planning to register in Finland as the Research Software Engineers Ry.
+The Nordic RSE group is planning to register in Finland as the
+Nordic Research Software Engineers Ry.
 Registering as an association comes with certain legal benefits, such as the ability to
 sign contracts and own assets.
 Registering an association will also create a clear membership and governing
 procedures.
 
-The association is governed by an annual assembly of its members, which is held
+The association is governed by an annual assembly of it's members, which is held
```
Comment (bast) [inline content/association/_index.md:20]:
```suggestion
The association is governed by an annual assembly of its members, which is held
```

File: content/association/_index.md [hunk 3]
```diff
@@ -5,37 +5,119 @@ title = "Association"
...
+An annual membership fee is likely to be necessary to cover costs such as the
+domain name and bank account.
+The member's annual meeting will decide the size of the membership fee.
 
 # Statutes
 
-The statutes of the association can be found [here](/association/statutes).
+Below is the proposal for the statutes of the association. While the Finnish
+translation will be official (if the association is registered in Finland,)
+the English translation is equivalent.
```
Comment (lucaferranti) [inline content/association/_index.md:37]: should we double check with an authorized translator that the translations are actually equivalent from a legal perspective?

**Comment by lucaferranti** [comment]
ah my review didn't make it in time 😄

**Comment by bast** [comment]
Sorry Luca that I merged there too quick.

**Pull request merged**